materials/tokens/sizes.mod.css

```diff
@@ -22,4 +24,11 @@
 
   /* Height */
   --token-size-height-input: 32px;
+
+  /* ==========================================================================
+   Tags
+   ========================================================================== */
+
+  /* Height */
+  --token-size-height-tags: 26px;
```

Comment:
We have `--token-size-height-input` with  `input` being singular.
We now add `--token-size-height-tags` with `tags` being plural.
We seem to use `--token-border-radius-tags` in plural as well though.

Can you elaborate why you went for plural for tags but singular for inputs?